### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#1623] [Announcements]
The 2.1.0 release is a major `lts` maintenance release

<hr /><em>This discussion was created from the release <a href='https://github.com/axmolengine/axmol/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by aismann** [comment]
> The 2.1.0 release is a major `lts` maintenance release
@halx99 
Maybe rename branch to: `2.1.0_LTS` is little bit more readable/information?

**Reply by halx99** [reply]
I make branch `2.1` as default branch, that's enough to let anyone known we will working on it long time

**Reply by aismann** [reply]
That's right.
